src/data-browser/maps/MapContainer.jsx

```diff
@@ -154,6 +151,8 @@ const MapContainer = props => {
   const [state2019Data, setState2019Data] = useState(null)
```

Comment:
Perhaps a refactor might be useful here. My thoughts are rather than having 8 different states perhaps convert it to two states where you have county data and state data. The states would both be objects and the keys are related to the year.

```javascript
const [countyData, setCountyData] = useState({
   2018: {...data},
   2019: {...data},
   2020: {...data},
   2021: {...data}
})
```
and

```javascript
const [stateData, setStateData] = useState({
   2018: {...data},
   2019: {...data},
   2020: {...data},
   2021: {...data}
})
```

Reply:
Created a separate ticket to follow up on this: https://github.com/cfpb/hmda-frontend/issues/1451